Updates problem-builder to v2.6.1

pomegranited opened the pull request:
Updates problem-builder to v2.6.1.
Ref https://github.com/open-craft/problem-builder/pull/128

**JIRA tickets**: Fixes the instructor_task import errors from [TNL-5903](https://openedx.atlassian.net/browse/TNL-5903).

**Dependencies**: https://github.com/edx/xblock-configuration/pull/9

**Sandbox URL**:
* LMS: http://pr13938-3.sandbox.opencraft.hosting
* Studio: http://studio-pr13938-3.sandbox.opencraft.hosting

**Partner information**: DavidsonX

**Testing instructions**:

1. Login as staff to sandbox: http://pr13938-3.sandbox.opencraft.hosting/
1. Navigate to `Features > Instructor Tool`
1. Ensure that you can filter and download results.
1. Smoke test the other Problem Builder features (should be unaffected).

**Reviewers**
- [x] @haikuginger 
- [ ] @dianakhuang 
- [ ] @BenjiLee 

**Settings**
```yaml
EDXAPP_EXTRA_REQUIREMENTS:
  - name: git+https://github.com/open-craft/problem-builder.git@v2.6.1#egg=xblock-problem-builder==2.6.1
```

edx-webhook commented:
Thanks for the pull request, @pomegranited! It looks like you're a member of a company that does contract work for edX. If you're doing this work as part of a paid contract with edX, you should talk to edX about who will review this pull request. If this work is not part of a paid contract with edX, then you should ensure that there is an OSPR issue to track this work in [JIRA](https://openedx.atlassian.net), so that we don't lose track of your pull request.

To automatically create an OSPR issue for this pull request, just visit this link: https://openedx-webhooks.herokuapp.com/github/process_pr?number=13938&repo=edx%2Fedx-platform

pomegranited commented:
jenkins run a11y

pomegranited commented:
jenkins run bokchoy

pomegranited commented:
jenkins run bokchoy

haikuginger commented:
@pomegranited, everything seems to be working, except that I can't seem to download CSV reports - get a 404. I went back and double-checked on my sandbox, and it's not working there either - I must have missed a step last night. Can you confirm that behavior/check for regression from the previous version?

pomegranited commented:
@haikuginger The issue with downloading report files is just a configuration issue with our OpenStack sandboxes, which will not be present in production.

I've fixed it for this instance, and will raise a ticket to fix it permanently.

haikuginger commented:
Excellent; thanks!

:+1:

- [x] I tested this: followed the instructions on the ticket and verified behavior
- [x] I read through the code
- [x] ~~I checked for accessibility issues~~ N/A; no UI changes
- [x] ~~Includes documentation~~ N/A

pomegranited commented:
Thanks, @haikuginger !

@BenjiLee Are you willing to do a hotfix release for us?  We'll also need some xblock-configuration changes, so I'll submit a DEVOPs ticket for that.

pomegranited commented:
@dianakhuang Are you willing to review this PR?  It addresses the issues raised in [TNL-5903](https://openedx.atlassian.net/browse/TNL-5903).

* The PR is rebased from master.
* The sandbox is a fresh deployment.
* The problem-builder xblock was deployed from the new tag, `v2.6.1`, using the `EDXAPP_EXTRA_REQUIREMENTS` string given above.

dianakhuang commented:
The updated code looks good to me. 👍

BenjiLee commented:
@pomegranited, @dianakhuang will be running the hotfix for this since I have no context for this change.

@dianakhuang If you need any release mastery support running the hotfix, let me know.

pomegranited commented:
Lovely, thank you @dianakhuang and @BenjiLee !
Btw I'm in Australia, so I'm on GMT+10:30, but I'll try to respond as quickly as possible.

pomegranited commented:
Wow ok, that got deployed really fast!  Sorry I missed your hotfix release email, @BenjiLee , but thank you and @dianakhuang for getting that out!
